lib/Settings/BeforePreferenceSetEventListener.php

```diff
@@ -76,6 +76,11 @@ public function validatePreference(string $userId, string $key, string|int|null
 			return $valid;
 		}
 
+		// "list-style" 'two-lines' / 'compact'
+		if ($key === UserPreference::CONVERSATIONS_LIST_STYLE) {
+			return $value === 'two-lines' || $value === 'compact';
```

Comment:
should compare to the constants `UserPreference::CONVERSATION_LIST_STYLE_…` but we can do it in a follow up